privacy/enforcement.go

```diff
@@ -9,11 +9,17 @@ type Enforcement struct {
 	GDPRGeo bool
 	GDPRID  bool
 	LMT     bool
+
+	// activities
+	UFPD       bool
+	Eids       bool
+	PreciseGeo bool
+	TID        bool
```

Comment:
This approach is hacky, but I'm ok with it because it will be short lived. In the near future, the Scrubber will have methods that align with activities and use the RequestWrapper to avoid inefficiency. I have other suggestions in this PR to try and improve worst case efficiency a bit for now.

Reply:
I agree, it's hacky. It should be changed to RequestWrapper.